docs/source/migration.mdx

```diff
@@ -1070,7 +1070,58 @@ By default, Apollo Server 4 uses a bounded in-memory cache backend (which _is sa
 
 For examples of using both `KeyvAdapter` and `InMemoryLRUCache`, see [Configuring external caching](/apollo-server/performance/cache-backends#configuring-external-caching).
 
-### Top-level error handling changes
+### Error handling changes
+
+#### ApolloError removal
+
+`ApolloError` together with `toApolloError` were removed in favor of using `GraphQLError` directly.
+`GraphQLError` is exported by `graphql` package and you can use it in your code:
```

Comment:
The example shows it but we should probably be explicit that you need to provide `extensions` and `code` the way you write it here rather than as top-level arguments to the constructor. Maybe we can show a before and after.

Reply:
Address in fd2c376708670269ed4a30103c57cd825e723f9f